OrbitClientGgp/ClientGgp.cpp

```diff
@@ -90,9 +95,11 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
   LOG("Capture pid %d", pid);
   TracepointInfoSet selected_tracepoints;
 
-  ErrorMessageOr<void> result =
-      capture_client_->StartCapture(thread_pool, pid, target_process_->GetName(), target_process_,
-                                    selected_functions, selected_tracepoints);
+  absl::flat_hash_map<std::string, ModuleData*> memory_map;
+  memory_map[main_module_->file_path()] = main_module_.get();
+
+  ErrorMessageOr<void> result = capture_client_->StartCapture(
+      thread_pool, *(target_process_.get()), memory_map, selected_functions, selected_tracepoints);
```

Comment:
Why do you pass `*(target_process_.get())` instead of providing `target_process_` directly?

Reply:
`target_process_` is now of the type `unique_ptr<ProcessData>` and the `StartCapture` takes `const ProcessData&`. So `*(target_process_.get())` does this conversion. I first changed the signature of the `StartCapture` method and then your `target_process_ from a shared_ptr to unique_ptr, so this conversion was the last step necessary. 

While I am looking at this, I just realized there is no real need to use `unique_ptr<ProcessData> target_process_` instead of a simple `ProcessData target_process_`. I think I am going to change it.